### DISCUSSION TITLE
Velero backup failed and when trying to check the logs its hang

### DISCUSSION THREAD

**Discussion by SurRepo** [vmware-tanzu/velero#5638] [Community support Q&A]
Hi all,

I tried to take one test ns backup using the below statement

**velero backup create testbkp --include-namespaces test**

**Then when I tried to describe the backup its showing Failed**

[admin@ip-192-168-76-2 ~]$ velero backup describe testbkp
Name:         testbkp
Namespace:    velero
Labels:       velero.io/storage-location=default
Annotations:  velero.io/source-cluster-k8s-gitversion=v1.21.14-eks-fb459a0
              velero.io/source-cluster-k8s-major-version=1
              velero.io/source-cluster-k8s-minor-version=21+

Phase:  Failed (run `velero backup logs testbkp` for more information)

Errors:    0
Warnings:  0

Namespaces:
  Included:  eocecm22
  Excluded:  <none>

Resources:
  Included:        *
  Excluded:        <none>
  Cluster-scoped:  auto

Label selector:  <none>

Storage Location:  default

Velero-Native Snapshot PVs:  auto

TTL:  720h0m0s

Hooks:  <none>

Backup Format Version:  1.1.0

Started:    2022-11-27 14:41:36 +0000 UTC
Completed:  <n/a>

Expiration:  2022-12-27 14:41:36 +0000 UTC

Velero-Native Snapshots: <none included>

**Now when I tried to check the logs, its just hang.** 
[admin@ip-192-168-76-2 ~]$ velero backup logs testbkp

No clue about the issue. Need support. Please help.

**Comment by qiuming-best** [comment]
@SurRepo what is the error output by velero backup logs testbkp when timeout

**Comment by SurRepo** [comment]
It’s just hang. Not getting any output after even 10 minutes.RegardsSurajit Sent from my iPhoneOn 28-Nov-2022, at 7:36 AM, qiuming ***@***.***> wrote:﻿
@SurRepo what is the error output by velero backup logs testbkp when timeout

**Reply by qiuming-best** [reply]
Maybe you should check the connection to the S3 bucket and permission of access to the bucket

**Comment by SurRepo** [comment]
Hi team,
Thanks for your response. Issue has been resolved. Issue was permission.

Regards,
Surajit
